test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
That IS confusing. On current master, or on this branch when I revert tc_basic.rb to current master, the three tests that I had changed fail:

```
[dave hladik:~/data/projects/activerecord-mysql2spatial-adapter] be rake test
Loaded testcase test/tc_basic.rb
Loaded testcase test/tc_spatial_queries.rb
Run options: --seed 35120

# Running:

.F.....F.F....

Finished in 3.045653s, 4.5967 runs/s, 8.5368 assertions/s.

  1) Failure:
RGeo::ActiveRecord::Mysql2SpatialAdapter::Tests::TestBasic#test_save_and_load_point_from_wkt [test/tc_basic.rb:141]:
No visible difference in the RGeo::Cartesian::PointImpl#inspect output.
You should look at the implementation of #== on RGeo::Cartesian::PointImpl or its members.
#<RGeo::Cartesian::PointImpl:0xXXXXXX "POINT (1.0 2.0)">

  2) Failure:
RGeo::ActiveRecord::Mysql2SpatialAdapter::Tests::TestBasic#test_readme_example [test/tc_basic.rb:165]:
Expected: true
  Actual: false

  3) Failure:
RGeo::ActiveRecord::Mysql2SpatialAdapter::Tests::TestBasic#test_set_and_get_point_from_wkt [test/tc_basic.rb:117]:
No visible difference in the RGeo::Cartesian::PointImpl#inspect output.
You should look at the implementation of #== on RGeo::Cartesian::PointImpl or its members.
#<RGeo::Cartesian::PointImpl:0xXXXXXX "POINT (1.0 2.0)">

14 runs, 26 assertions, 3 failures, 0 errors, 0 skips
```

When I constrain rgeo to 0.4.* these tests pass for me.